Documentation/building/windows/instructions.md

```diff
@@ -18,15 +18,19 @@ MSBuild version 15 or later is required.
 
  4. (Optional) [Configure the build](../configuration.md).
 
- 5. (Optional) In a [Developer Command Prompt][developer-prompt], prepare external git dependencies:
+ 5. In a [Developer Command Prompt][developer-prompt], build the preparation utility:
+ 
+        msbuild build-tools/xaprepare/xaprepare.sln
+ 
+ 6. (Optional) In a [Developer Command Prompt][developer-prompt], prepare external git dependencies:
 
-        msbuild Xamarin.Android.sln /t:PrepareExternal
+        build-tools/xaprepare/xaprepare/Debug/xaprepare /s:PrepareExternalGitDependencies
 
     This will configure external proprietary components such as monodroid.
 
  6. In a [Developer Command Prompt][developer-prompt], prepare the project:
 
-        msbuild Xamarin.Android.sln /t:Prepare
+        build-tools/xaprepare/xaprepare/Debug/xaprepare
```

Comment:
My question would be, can we just make `msbuild Xamarin.Android.sln /t:Prepare` do this step? and then we don't have to add a new step to building Xamarin.Android? (or change the docs at all 😄)

Reply:
Let's see if we can just do this instead: https://github.com/xamarin/xamarin-android/pull/3185

And if it doesn't work our for some reason, we can revisit this PR.

Reply:
We should definitely be able to wrap whatever `xaprepare.exe` needs to do in the `/t:Prepare` target.

Reply:
`Prepare` already performs the step, I updated this PR accordingly